blocks/components/editable/index.js

```diff
@@ -57,6 +59,28 @@ export default class Editable extends wp.element.Component {
 		this.props.onChange( value );
 	}
 
+	onKeyDown( event ) {
+		if ( ! this.props.tagName && event.keyCode === 13 ) {
+			// Wait for the event to propagate
+			setTimeout( () => {
+				// Getting the content before and after the cursor
+				this.editor.selection.getStart();
+				const childNodes = Array.from( this.editor.getBody().childNodes );
+				const splitIndex = childNodes.indexOf( this.editor.selection.getStart() );
+				const getHtml = ( nodes ) => nodes.reduce( ( memo, node ) => memo + node.outerHTML, '' );
+				const beforeNodes = childNodes.slice( 0, splitIndex );
+				const before = getHtml( beforeNodes );
+				const after = getHtml( childNodes.slice( splitIndex ) );
+
+				// Splitting into two blocks
+				this.editor.setContent( this.props.value );
```

Comment:
Yes, that's the intent. By this, I revert the content to the previous state before hitting Enter, because this change is "controlled", the split callback should update the content accordingly.

If I drop this line, it will work for the moment, but I know that without it we can have issues (Forgot exactly why, this is comming from the multi-instance). I think maybe focus issues.